### PR TITLE
feat: replace `cozy-cloud` icon by `share` icon on shared ciphers

### PR DESCRIPTION
**Pull request by Ldoppea**
Before:
image

After:
image

### CODE REVIEW

File: src/scss/cozy.scss [hunk 1]
```diff
@@ -77,6 +77,14 @@ app-vault-groupings {
         display: flex;
         margin-right: 5px;
 
+        svg {
+            fill: var(--coolGrey);
```
Comment (Crash--) [inline src/scss/cozy.scss:81]: semantic? :D

**Review by Crash-- — APPROVED** [review]
Alignment and size seems still weird but 👍

File: src/scss/cozy.scss (hunk 1 above)
Comment (Ldoppea) [reply src/scss/cozy.scss]: Fixed

**Comment by Ldoppea** [comment]
> Alignment and size seems still weird but 👍

I agree, I didn't take too long on the UI as the design will be changed soon.

**Pull request merged**